tmt/steps/__init__.py

```diff
@@ -19,6 +19,7 @@
 from tmt.options import show_step_method_hints
 
 if TYPE_CHECKING:
+    import tmt
```

Comment:
@janhavlin this seems incorrect, `tmt` import is also a runtime dependency. See Build logs for errors which flap on this:
https://download.copr.fedorainfracloud.org/results/packit/teemtee-tmt-1333/epel-8-x86_64/04685763-tmt/builder-live.log.gz

Reply:
Tried to import it so I can use `tmt.Test`. Ok, changed it to `from tmt.base import Test`.